node/service/src/lib.rs

```diff
@@ -801,7 +801,7 @@ where
 	let auth_or_collator = role.is_authority() || is_collator.is_collator();
 	let requires_overseer_for_chain_sel = local_keystore.is_some() && auth_or_collator;
 
-	let pvf_checker_enabled = !is_collator.is_collator() && chain_spec.is_versi();
+	let pvf_checker_enabled = role.is_authority() && !is_collator.is_collator();
```

Comment:
In order to submit votes a node is required to be an active validator, enabling it otherwise will unnecessarily poll runtime at each activation.